internal/appsec/emitter/httpsec/http.go

```diff
@@ -111,14 +110,18 @@ func ExecuteSDKBodyOperation(parent dyngo.Operation, args SDKBodyOperationArgs)
 	return err
 }
 
+type Span interface {
+	SetTag(key string, value interface{})
+}
```

Comment:
This would be a synonym to `trace.TagSetter` I believe... So that's not needed?

Reply:
Looks like it, yeah.